teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +122,283 @@ requirements, and provenance attestations can be made to meet software supply ch
+   [TEP-0060: Remote Resource Resolution][tep-0060]. 
+   The verification is discussed further [below](#verified-remote-resources-1).
+6. The resource must be well documented with all configuration options described 
+   and a working example provided.
+7. The resource must follow and demonstrate best practices, such as the 
+   [`Task` authoring recommendations][task-authoring-recommendations].
+
+#### Community and Official Catalogs in the Tekton Hub
+
+Users rely on the [Tekton Hub][hub] to discover shared resources. 
+The Tekton Hub supports publishing resources from multiple Catalogs.
+As described in [TEP-0003][tep-0003-hub], the goal was to set it up to
+support providing support tiers for shared resources. It also allows
+users and organizations to create their own Catalogs and share them in
+the Tekton Hub, as long as they comply with the Catalog contract. The
+Tekton Hub can indicate the source Catalog of each resource, such as
+through a badge or tag. 
+
+To add the Official Catalog, we only have to modify the Catalogs 
+[configuration][hub-config] in the Hub, as such:
+
+```yaml
+catalogs:
+  
+  # community catalog
+  - name: tekton
+    org: tektoncd
+    type: community
+    provider: github
+    url: https://github.com/tektoncd/catalog
+    revision: main
+    
+  # official catalog  
+  - name: tekton-official
+    org: tektoncd
+    type: official
+    provider: github
+    url: https://github.com/tektoncd/catalog-official
+    revision: main
+```
+
+This provides extensibility to the support tiers: we can add additional
+Catalogs for other support tiers as needed in the future. 
+
+#### Responsibilities
+
+##### Resource Ownership
+
+The responsibilities of owning a resource include but are not limited to:
+
+1. Reviewing and approving all changes to the resource.
+2. Resolving any failures of tests validating the resource.
+3. Responding to and resolving any issues reported concerning the resource.
+4. Updating the resources to be compatible with new versions of Tekton, and
+   other dependencies.
+
+##### Catalog Ownership
+
+The responsibilities of owning a Catalog include but are not limited to:
+
+1. Reviewing and approving new resources contributed to the Catalogs.
+2. Maintaining the health of the testing infrastructure.
+3. Triaging issues and involving the owners of the affected resources.
+4. Ensuring the resources meet the quality standards of the Catalog.
+
+#### Design Evaluation
+
+The Catalogs will leverage existing structure and infrastructure that 
+supports the existing `tektoncd/catalog`. Decoupling the support tiers 
+into separate repositories makes it easy to distinguish the support tiers, 
+enforce the applicable quality standards, and leverage existing structure 
+and infrastructure. Moreover, reusing the existing Catalog at 
+`tektoncd/catalog` minimizes effort to provide the support tiers.
+
+#### Alternatives
+
+##### 1. One Repository and Use Annotations
```

Comment:
This feels like my preferred option, but happy to discuss further on it.

Reply:
The problem with this approach is that I can come, and add a new task with the official annotation, and if it gets missed in the review, we end-up supporting something that we weren't supposed to.

Reply:
Well, that's what reviews are for :) 
I still have a feeling that having multiple repos might gives us more overhead and less flexibility for a weak benefit - but   I'm not feeling strongly about it so let's see how this goes.  I think we agreed during the WG yesterday to go for the multi-repo option.

Reply:
thanks for the discussion at the catalog wg :)